release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -195,6 +195,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 [id="ocp-4-9-sandboxed-containers"]
 === {sandboxed-containers-first}
 
+FIPS mode is now automatically enabled for {sandboxed-containers-first}. {sandboxed-containers-first} deployed on an {product-title} cluster installed in FIPS mode will not taint the cluster's FIPS support. For more information, see
+
+The {sandboxed-containers-operator} now includes a must-gather image, allowing you to collect custom resources and log files specific to this operator and the underlying runtime components for diagnostic purposes. For more information, see
```

Comment:
Would it make sense to break these paragraphs out as either H4s or a bulleted list?